deluge/ui/gtkui/path_combo_chooser.py

```diff
@@ -412,17 +422,17 @@ def on_stored_values_treeview_key_release_event(self, widget, event):
 
         """
         keyval = event.keyval
-        ctrl = event.state & gtk.gdk.CONTROL_MASK
+        ctrl = event.get_state() & Gdk.ModifierType.CONTROL_MASK
 
         # Edit selected row
-        if (keyval in [keysyms.Left, keysyms.Right, keysyms.space]):
+        if (keyval in [Gdk.KEY_Left, Gdk.KEY_Right, Gdk.KEY_space]):
             path = self.get_selection_path()
             if path:
                 self.on_edit_path(path, self.tree_column)
         elif key_is_up_or_down(keyval):
             # Swap the row value
-            if event.state & gtk.gdk.CONTROL_MASK:
-                self.handle_list_scroll(_next=key_is_down(keyval),
+            if event.get_state() & Gdk.ModifierType.CONTROL_MASK:
+                self.handle_list_scroll(next=key_is_down(keyval),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected keyword argument 'next' in method call](https://www.codacy.com/app/tbkizle/deluge/file/2628318269/issues/source?bid=3238201&fileBranchId=3238205#l435)